core/src/main/java/org/pentaho/platform/scheduler2/action/IActionRunner.java

```diff
@@ -55,4 +55,10 @@ public interface IActionRunner extends Callable<Boolean> {
    */
   void setStreamProvider( IBackgroundExecutionStreamProvider streamProvider );
 
+  /**
```

Comment:
This is method interface is not used, only I would delete` IActionRunner#addJcrParams`.

No enterprise code uses this method as of now.